packages/forms/src/styled/common/StyledLegend.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { hideVisually } from 'polished';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'forms.fieldset_legend';
+
+interface IStyledLegend {
+  isCompact?: boolean;
+  isHidden?: boolean;
```

Comment:
Remove. The correct usage is via the `hidden` attribute, as supported by all other `Label` components. See https://garden.zendesk.com/components/radio#hidden-label for details.